src/common/utils/channel/message-channel-injection-token.ts

```diff
@@ -4,9 +4,9 @@
  */
 
 import { getInjectionToken } from "@ogre-tools/injectable";
-import type { JsonValue } from "type-fest";
+import type { IpcValue } from "./allowed-types";
 
-export interface MessageChannel<Message extends JsonValue | void = void> {
+export interface MessageChannel<Message extends IpcValue | void = void> {
```

Comment:
Unit tests for new types are missing.

Reply:
Why do we need to unit test a dependency?

Reply:
Discussed elsewhere